Mimecast/Mimecast.json

```diff
@@ -1,6 +1,6 @@
 {
     "type": "FolderSyncDefinition",
-    "name": "Mimecast",
+    "name": "Mimecast App for Sumo Logic",
     "description": "Dashboards for Mimecast Secure Email Gateway",
```

Comment:
Probably update the description here to reflect the developer, like "developed by Mimecast Team"?